<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showWindow</class>
 <widget class="QMainWindow" name="showWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Results</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icons/gar-fish.ico</normaloff>:/icon/icons/gar-fish.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelVid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>611</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plotLayout"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
